2023/NorthTorontoUG/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "NorthTorontoUGAdmin"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "North Toronto Cloud & DevOps User Group",
+    "Country": "Canada",
+    "Url": "https://www.meetup.com/northtorontoug/events/291512241/",
+    "SessionizeApiId": "",
```

Comment:
You're not using Sessionize to show off your sessions on the Global Azure web site?